src/protagonist/API/Features/Image/ImageController.cs

```diff
@@ -162,6 +178,19 @@ public async Task<IActionResult> GetImage(int customerId, int spaceId, string im
             return this.HydraProblem("Delivery channels are disabled", assetId.ToString(), 400, "Bad Request");
         }
         
+        if (apiSettings.EmulateOldDeliveryChannelProperties && 
+            hydraAsset.WcDeliveryChannels != null)

```

Comment:
Line 175 uses `!hydraAsset.WcDeliveryChannels.IsNullOrEmpty()` but this uses `hydraAsset.WcDeliveryChannels != null` - they should use the same logic? Since they both use this same check can they be combined?